packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1571,6 +1571,10 @@ class Padding extends SingleChildRenderObjectWidget {
 /// widget positions the `child` such that both points are lined up on top of
 /// each other.
 ///
+/// The following images illustrate different uses of the Align class.
+///
+/// ![A blue square container with the Flutter logo in the top right corner.](https://flutter.github.io/assets-for-api-docs/assets/widgets/align_constant.png)
```

Comment:
Any change we can move the images inside the `{@tool sample}~ clauses that they are illustrating?